program/src/state.rs

```diff
@@ -22,6 +22,7 @@ pub const SEED_PREFIX_GOVERNANCE: &[u8] = b"governance";
 pub const SEED_PREFIX_VOTE: &[u8] = b"vote";
 
 /// Derive the address of the treasury account.
+// Jon: this will likely need the governance address too
```

Comment:
The doc only specifies a single treasury (piggy bank), and it also just says the system will distribute to "stakers" and "holders", so if we have multiple treasuries, how are we expected to distribute rewards amongst them?

Reply:
I meant one treasury per governance, assuming the system needs to handle multiple governance

Reply:
Right but at the top-level of the overall system, there's a reward funnel that aims to distribute 10% of rewards to the "piggy bank", then 40% to stakers and 50% to holders. With one piggy bank, one staking ecosystem, and a community of holders this is very trivial to do. With multiple layers for governance and staking, it's not as straightforward to determine how to send, say that 10% slice of the pie to the treasur(ies).

Reply:
There's still only one governance and treasury as far as PAL is concerned, but this program allows for the creation of more than one governance. The funnel program only cares about one governance / treasury, which we configure in that program

Reply:
Done in #10.